tests/acceptance/features/apiSharingNg1/listPermissions.feature

```diff
@@ -1885,3 +1885,1095 @@ Feature: List a sharing permissions
         }
       }
       """
+
+  @issues-8428
+  Scenario: user lists permissions of a shared folder in personal space
+    Given user "Brian" has been created with default attributes and without skeleton files
+    And user "Alice" has created folder "folder"
+    And user "Alice" has sent the following resource share invitation:
+      | resource        | folder   |
+      | space           | Personal |
+      | sharee          | Brian    |
+      | shareType       | user     |
+      | permissionsRole | Viewer   |
+    And user "Alice" has created the following resource link share:
+      | resource        | folder   |
+      | space           | Personal |
+      | permissionsRole | view     |
+      | password        | %public% |
+    When user "Alice" gets permissions list for folder "folder" of the space "Personal" using the Graph API
+    Then the HTTP status code should be "200"
+    And the JSON data of the response should match
+      """
+      {
+        "type": "object",
+        "required": [
+          "@libre.graph.permissions.actions.allowedValues",
+          "@libre.graph.permissions.roles.allowedValues",
+          "value"
+        ],
+        "properties": {
+          "@libre.graph.permissions.actions.allowedValues": {
+            "type": "array",
+            "enum": [
+              [
+                "libre.graph/driveItem/permissions/create",
+                "libre.graph/driveItem/children/create",
+                "libre.graph/driveItem/standard/delete",
+                "libre.graph/driveItem/path/read",
+                "libre.graph/driveItem/quota/read",
+                "libre.graph/driveItem/content/read",
+                "libre.graph/driveItem/upload/create",
+                "libre.graph/driveItem/permissions/read",
+                "libre.graph/driveItem/children/read",
+                "libre.graph/driveItem/versions/read",
+                "libre.graph/driveItem/deleted/read",
+                "libre.graph/driveItem/path/update",
```

Comment:
these are already covered in above scenarios. for these scenarios, we can `value` in details